framework/src/quake_camera.cpp

```diff
@@ -2,12 +2,16 @@
 
 namespace gvk
 {
-	quake_camera::quake_camera()
+	quake_camera::quake_camera(std::string aName)
 		: mRotationSpeed(0.001f)
 		, mMoveSpeed(4.5f) // 4.5 m/s
 		, mFastMultiplier(6.0f) // 27 m/s
 		, mSlowMultiplier(0.2f) // 0.9 m/s
 	{
+		if (!aName.empty())
+		{
+			set_name(aName);
+		}
```

Comment:
```suggestion
		set_name(std::move(aName));
```

Also here, a move would be the most efficient option. No need for the `.emtpy()` check!

Reply:
Even better would be to _not_ invoke `set_name`, but instead, invoke the base class' constructor:

```
invokee::invokee(std::string, bool, int)
```

Reply:
I'll add a suggestion above